pyseir/icu/utils.py

```diff
@@ -28,10 +28,13 @@ def calculate_case_based_weights() -> dict:
     last_month_cum_cases.name = "summed_cases"
 
     df = last_month_cum_cases.reset_index().dropna()
-    df["state_fips"] = df["fips"].str.slice(0, 2)  # Apply State Labels to Groupby
+    # Example location_id value = 'iso1:us#iso2:us-ak#fips:02013'
+    df["state_location_id"] = df["location_id"].str.split("#").str[1]  # 'iso2:us-ak'
+    df["fips_location_id"] = df["location_id"].str.split("#").str[-1].str.split(":").str[-1]
```

Comment:
What depends on this value? It looks like the column name in `output` changed from `fips`, what about the reader?

Reply:
Both "state_location_id" and "fips_location_id" are internal to this function. I switched to CommoFields.FIPS, but have to give more thought about generalizing this to move beyond the state-county mapping.